docs/source/tutorial/peft_integrations.md

```diff
@@ -28,7 +28,7 @@ For example, load a base model and then load the [artificialguybr/3DRedmond-V1](
 
 ```py
 import torch
-from diffusers import DiffusionPipeline
+from diffusers.pipelines.pipeline_utils import DiffusionPipeline
```

Comment:
No this is not required. Could you help us with a Colab that reproduces this issue?

Reply:
Hi, thank you for the attention. Indeed it does not seem necessary. I'm sorry for the oversight. However I get this from Pyright language server when using the original import:

<img width="1183" alt="image" src="https://github.com/huggingface/peft/assets/39929801/8bd52db4-66e2-49c4-a238-f47d7a32063c">

This kind of warning makes it impossible to pass the training script through CI. Do you know why that happens?

Reply:
The reason why pyright complains is probably because it does not understand the lazy import mechanism used in diffusers: https://github.com/huggingface/diffusers/blob/main/src/diffusers/__init__.py

This should probably be solved by telling pyright to ignore this line (probably `# type: ignore`). Anyway, since this is just relevant for your script, I would suggest to close this PR (still thanks for trying to fix this!).

Reply:
Thank you all for the support.